sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -21,6 +24,10 @@ public abstract class RecordedTestBase : ClientTestBase
 
         public TestRecording Recording { get; private set; }
 
+        private static TimeSpan ZeroPollingInterval { get; } = TimeSpan.FromSeconds(0);
+
+        protected ResourceGroupCleanupPolicy CleanupPolicy { get; set; }
```

Comment:
Does this property have to be in the `RecordedTestBase`? It's only initialized in the generic version of the class?